calculator.rb

```diff
@@ -14,6 +14,11 @@ def divide(operand_1, operand_2)
   return operand_1 / operand_2
 end
 
+# Raise operand_1 to the operand_2th power
```

Comment:
This comment explains _what_, not _why._ It's not a very good comment. I think we should remove it.